runtime/voxel.c

```diff
@@ -26,6 +26,13 @@ int main(int argc, char* argv[]) {
 
     fseek(fp, 0, SEEK_SET);
     fread(data, sizeof(char), size, fp);
```

Comment:
Looks like you forgot to remove this line — it's causing `fread` to read twice, which is causing tests to fail!